src/main/resources/api/service_openapi.yaml

```diff
@@ -155,6 +155,28 @@ paths:
         200:
           $ref: '#/components/responses/QueryResponse'
 
+  /api/v1/unique-values/{version}:
+    post:
+      summary: Returns all unique values
+      description: Return unique values given a qualified columnName
+      operationId: uniqueValues
+      tags:
+        - query
+
+      parameters:
+        - $ref: '#/components/parameters/DatasetVersion'
+
+      requestBody:
+        description: TABLE.column_name of table value being requested
```

Comment:
The input doesn't contain `TABLE`, it only contains column names, e.g., `column.column.column`.

Reply:
FIxed this.   
Originally thought we would need inspection of tables beyond the cda_mvp